python/ray/tests/test_advanced_3.py

```diff
@@ -696,6 +697,25 @@ def test_ray_address_environment_variable(ray_start_cluster):
     ray.shutdown()
 
 
+def test_gpu_info_parsing():
+    info_string = """Model:           Tesla V100-SXM2-16GB
```

Comment:
Can you add a couple more info strings for testing? Doesn't need to include all the stuff just the model line